email_notification/helpers.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+from permissions.helpers import has_unit_permission
+from reservations.models import Reservation
+from spaces.models import Unit
+from users.models import ReservationNotification, User
+
+
+def get_notification_recipients(
+    reservation: Reservation, notification_settings: List[ReservationNotification]
+) -> List[str]:
+    notification_recipients: List[User] = []
+    reservation_units = reservation.reservation_unit.all()
+    units = Unit.objects.filter(reservationunit__in=reservation_units)
+
+    # Get all users and collect the ones that can manage relevant units,
```

Comment:
We could make this comment a docstring `""" get all users.... """` and put this as a first line of the function.

Reply:
Changed 👍